boring/src/ssl/mod.rs

```diff
@@ -1281,6 +1304,21 @@ impl SslContextBuilder {
         }
     }
 
+    /// Gets the list of supported ciphers for protocols before TLSv1.3.
+    ///
+    /// See [`ciphers`] for details on the format
+    ///
+    /// This corresponds to [`SSL_CTX_get_ciphers`].
+    ///
+    /// [`ciphers`]: https://www.openssl.org/docs/manmaster/man1/ciphers.html
+    /// [`SSL_CTX_set_cipher_list`]: https://www.openssl.org/docs/manmaster/man3/SSL_CTX_get_ciphers.html
+    pub fn get_ciphers(&mut self) -> Result<&StackRef<SslCipher>, ErrorStack> {
```

Comment:
I don't think the Result is necessary here

https://github.com/google/boringssl/blob/b628f8721e7e67302b0e26c92e0108a31066194e/ssl/ssl_lib.cc#L2075-L2077

Seems like NULL should be interpreted as None, or even maybe that the method should panic on NULL.

https://github.com/google/boringssl/blob/master/ssl/internal.h#L631